data/content/about.md

```diff
@@ -1,3 +1,51 @@
-Free and guided workshops in Rust – a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+RustBridge is a workshop focused on getting underrepresented people with
+a background in another programming language to learn Rust and join the
+community.
 
-Our events are aimed at people who are underrepresented in technology and offer the opportunity to learn within a friendly, safe and sometimes fun environment. We do this by providing rewarding learning experiences, building helpful tools as well as materials and gathering a group of awesome humans. RustBridge is a project by the Rust Language Community Team.
+### Why RustBridge?
+
+RustBridge is an excellent way to welcome and attract persons from
+underrepresented backgrounds to your Rust conference or event. By giving
+students new to Rust an introduction to the language and other fellow Rust
+learners before the conference, RustBridge increases the relevance and
+value of Rust event content for students, as well as offering students a
+sense of belonging and camaraderie, both of which are things new members
+to a community often lack. Students of previously run Bridge programs often
+cite the Bridge Program as the reason they felt comfortable attending the
+event, and that the program greatly increased their enjoyment of the event.
+
+### Why Rust?
+
+[Rust is a programming language] designed to improve the experience of systems
+programming. Systems programming is notoriously difficult and learners are
```

Comment:
how about "has a reputation for being difficult"? which is kind of what "notorious" means but i feel like it's the "is" that's not quite right here. 

I usually say something like it's not that Rust makes systems programming easy, there's still hard stuff, but Rust works with you to make writing safe systems code that works *possible* without needing to know about a bajillion footguns. footguns is jargon tho, idk how to say that betterrrr